doc/source/whatsnew/v1.5.0.rst

```diff
@@ -433,6 +433,7 @@ I/O
 Period
 ^^^^^^
 - Bug in subtraction of :class:`Period` from :class:`PeriodArray` returning wrong results (:issue:`45999`)
+- Bug in :meth:`Period.strftime` and :meth:`PeriodIndex.strftime`, directives `%l` and `%u` were giving wrong results (:issue:`46252`)
```

Comment:
I was not very sure if I should write this under "Period" or "Datetimelike", let me know.

Reply:
here is fine